notebooks/service/service.go

```diff
@@ -22,10 +22,10 @@ type NotebookSpec map[string]interface{}
 
 // NotebookService is the service contract for Notebooks.
 type NotebookService interface {
```

Comment:
The service contract is updated to match the new `notebooksd` service - see #21386